core/src/epicli/data/azure/validation/infrastructure/availability-set.yml

```diff
@@ -0,0 +1,22 @@
+required:
+  - availability_set
+  - platform_fault_domain_count
+  - platform_update_domain_count
+  - managed
+properties:
+  availability_set:
+    "$id": "#/properties/specification/properties/availability_set"
+    type: string
+    pattern: ^[a-z0-9\-_]+$
+  platform_fault_domain_count:
+    "$id": "#/properties/specification/properties/platform_fault_domain_count"
+    type: integer
+    minimum: 1
+  platform_update_domain_count:
+    "$id": "#/properties/specification/properties/platform_update_domain_count"
+    type: integer
+    minimum: 1
+  managed:
+    "$id": "#/properties/specification/properties/managed"
+    type: string
```

Comment:
Can we accept (switch to) YAML bool type for consistency with other config docs? https://yaml.org/type/bool.html

Reply:
Yes. I'll work on that!